web/src/portico/help.ts

```diff
@@ -80,15 +88,22 @@ function scrollToHash(simplebar) {
     }
 }
 
-const cache = new Map();
-const loading = {
+type Article = {
+    html: string;
+    title: string;
+};
```

Comment:
Probably a nit but we currently kind of follows a convention of putting all the type definitions at the top of the file, so maybe we should follow it here as well.

Reply:
I agree. good spot!